### PR TITLE
Multiple documentation fixes

### PR DESCRIPTION
**Pull request by englercj**
While working on a JSDoc3 theme meant to generate typescript definition files from JSDoc annotated JavaScript source (https://github.com/englercj/tsd-jsdoc) I found a few documentation errors throughout the source.

This PR is me fixing those, as well as replacing some tabs with spaces.

### CODE REVIEW

File: src/core/renderers/webgl/WebGLState.js [hunk 1]
```diff
@@ -2,63 +2,62 @@ var mapWebGLBlendModesToPixi = require('./utils/mapWebGLBlendModesToPixi');
```
Comment (englercj) [inline src/core/renderers/webgl/WebGLState.js]: This file was using a mix of tabs and spaces, so I normalized it to 4 spaces.

**Comment by ivanpopelyshev** [comment]
nice

File: src/core/sprites/canvas/CanvasTinter.js [hunk 2]
```diff
@@ -1,250 +1,268 @@
...
-    var texture = sprite.texture;
-
-    color = CanvasTinter.roundColor(color);
-
-    var stringColor = '#' + ('00000' + ( color | 0).toString(16)).substr(-6);
-
-    texture.tintCache = texture.tintCache || {};
-
-    if (texture.tintCache[stringColor])
+var CanvasTinter = module.exports = {
```
Comment (englercj) [inline src/core/sprites/canvas/CanvasTinter.js]: The end result of the export is the same as before, but now that it is formatted in this way it is cleaner and jsdoc is parsing it correctly.

**Comment by bigtimebuddy** [comment]
👍

**Comment by GoodBoyDigital** [comment]
Nice!

**Comment by bigtimebuddy** [comment]
@englercj I think this is good to merge.

**Pull request merged**